src/emmc.c

```diff
@@ -185,6 +186,10 @@ pu_emmc_setup_layout(PuFlash *flash,
             part->size = self->expanded_part_size;
         }
 
+	if (part->type == PED_PARTITION_LOGICAL) {
```

Comment:
Please add a comment above this if-statement that adding 2 sectors is needed because of the EBR, which is subtracted off of the size later on during partition creation.